packages/playwright-ct-core/src/vitePlugin.ts

```diff
@@ -90,14 +88,17 @@ export function createPlugin(
           outDir: use.ctCacheDir ? path.resolve(configDir, use.ctCacheDir) : path.resolve(templateDir, '.cache')
         },
         preview: {
-          https: baseURL.protocol.startsWith('https:'),
+          https: baseURL.protocol.startsWith('https:') ? {} : undefined,
```

Comment:
`preview.https` is dropped: https://vitejs.dev/guide/migration#remove-https-flag-and-https-true

Reply:
It is still there, but is no longer boolean. My change retains the behavior, but as per the deprecation message it starts https w/o a certificate. So we can probably remove it.